test/state/host.cpp

```diff
@@ -409,9 +415,7 @@ evmc::Result Host::execute_message(const evmc_message& msg) noexcept
     if (is_precompile(m_rev, msg.code_address))
         return call_precompile(m_rev, msg);
 
-    // In case msg.recipient == msg.code_address, this is the second lookup of the same address.
```

Comment:
Comment is still true I think.

Reply:
Kind of true, but now this is the `State::get_code()` API issue: how to combine local account lookup with code lookup.